### PR TITLE
Fix attribution HTML glyph reference and fix historical usages as well

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #https://github.com/WordPress/openverse-frontend/issues/857 by @sarayourfriend 

## Description

The folder structure needs to mirror the old path used by CC search, hence the nested `static` folders being added for this.

Test the attribution HTML, that it renders as expected.

We'll write an e2e test for this later to render the attribution HTML and visual-regression test it with Playwright but this needs to be deployed to production before that will work.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sarayourfriend** [comment]
Labeling as critical because all historical attribution HTML usages are currently broken because of this.

**Review by zackkrida — APPROVED** [review]
👍

**Review by krysal — CHANGES_REQUESTED** [review]
I guess we can't fully test this until deploying to production. It will work for the CC icon but all other license icons are needed (cc-by_icon, cc-nc_icon, etc.)

**Comment by sarayourfriend** [comment]
Thanks for catching that @krysal, I could have sworn I searched the module for `baseAssetsPath` and only found one usage :facepalm: I've updated the PR to include copies of each of the license part icons as well.

File: src/static/static/img/cc_icon.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg">
+  <!-- DO NOT MOVE OR RENAME THIS FILE OR HISTORICAL USAGES OF THE ATTRIBUTION HTML GENERATED BY THE APP WILL BREAK -->
```
Comment (krysal) [inline src/static/static/img/cc_icon.svg:2]: Should we add the same comment to the other icons or in a README in this folder?

File: src/static/static/img/cc-pdm_icon.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg">
+	<g id="icon" fill="currentColor">
+		<path d="M15 0C6.716 0 0 6.716 0 15c0 8.283 6.716 15 15 15 8.284 0 15-6.717 15-15 0-8.284-6.716-15-15-15zm0 27.187C8.28 27.187 2.812 21.72 2.812 15c0-1.407.243-2.758.683-4.015l4.767 2.122a10.081 10.081 0 00-.198 1.991c0 5.988 4.47 7.675 7.282 7.675a7.314 7.314 0 004.217-1.32c.214-.153.407-.307.585-.464l-2.187-2.657a2.496 2.496 0 01-.2.206c-.85.806-1.785.806-1.994.806-2.288 0-3.246-2.414-3.258-4.348l13.137 5.85c.015.002.028.002.041.005-2.073 3.774-6.086 6.336-10.687 6.336zm11.745-9.073l-13.55-6.048c.489-.885 1.267-1.55 2.431-1.55.675 0 1.21.223 1.632.506.175.126.327.252.456.38l2.442-2.514c-1.64-1.41-3.505-1.663-4.725-1.663-2.866 0-4.895 1.24-6.09 3.121L4.81 8.326C6.99 5.007 10.743 2.811 15 2.811c6.72 0 12.188 5.467 12.188 12.188 0 1.088-.145 2.14-.414 3.145-.011-.01-.018-.02-.03-.03z" />
+	</g>
+</svg>
```
Comment (krysal) [inline src/static/static/img/cc-pdm_icon.svg:5]: A new line here would be 👍

**Review by krysal — APPROVED** [review]
That was fast! LGTM.

**Comment by zackkrida** [comment]
@krysal `pnpm build && pnpm start` should be a solid way to test the building of the static assets. I'd definitely recommend doing so for testing this PR!

File: src/static/static/img/cc_icon.svg (hunk 1 above)
Comment (sarayourfriend) [reply src/static/static/img/cc_icon.svg:2]: Oh true! Maybe I'll just create a "DO_NOT_MOVE_THIS_DIRECTORY.md" with an explanation and we can somehow exclude it from Nuxt's static file serving so it doesn't end up getting crawled by some very curious crawler or something :joy:

Comment (sarayourfriend) [reply src/static/static/img/cc_icon.svg]: Turns out Nuxt will ignore hidden files in the static folder :+1: 

Added a very loud README in https://github.com/WordPress/openverse-frontend/pull/990/commits/2c5b05e65746a414427976f519ec61e1f12495ca

File: src/static/static/img/cc-pdm_icon.svg (hunk 2 above)
Comment (sarayourfriend) [reply src/static/static/img/cc-pdm_icon.svg]: Done in https://github.com/WordPress/openverse-frontend/pull/990/commits/2c5b05e65746a414427976f519ec61e1f12495ca

**Comment by zackkrida** [comment]
rereviewing with another LGTM!

**Pull request merged**